namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java

```diff
@@ -171,7 +171,7 @@ public void deleteTopic(final String topic, final String clusterName) {
             this.lock.writeLock().lockInterruptibly();
             //get all the brokerNames fot the specified cluster
             Set<String> brokerNames = this.clusterAddrTable.get(clusterName);
-            if (brokerNames == null || brokerNames.isEmpty()) {
+            if (brokerNames == null) {
```

Comment:
why remove this?
suggest `CollectionUtils.isEmpty(brokerName)`

Reply:
if (nameSet.isEmpty()) {
    this.clusterAddrTable.remove(clusterName);
    log.info("unregisterBroker, remove cluster from clusterAddrTable {}", clusterName);
}